planet-themes-plugin/css/neptune.css

```diff
@@ -162,3 +163,126 @@ IEclipsePreferences#org-eclipse-pde-ui:io-github-pyvesb-eclipse_planet_themes-ne
     'editor.color.tag=105,159,213'
     'editor.color.xml_comment=196,217,240';
 }
+
+
+/* Color Palette (This is informative, there is no way of interpolating this into preference tag) */
```

Comment:
If we don't find a way to make these work, I'd suggest removing them, they'll just be more things to load by Eclipse at startup. :)

Reply:
After discussing the matter with vogella, this will be removed in the next commit